resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -46,7 +46,7 @@
 	 * @returns {jQuery} Returns a jQuery promise
 	 * @see https://developers.facebook.com/docs/javascript/quickstart/v2.2
 	 */
-	$.loadFacebookAPI = function (callback) {
+	$.loadFacebookSDK = function (callback) {
```

Comment:
@armonr after looking into it, I think the placement of `$('.sso-login').removeClass('hidden')` should still be okay. It's inside of the `window.fbAsyncInit` function which is called by the FB SDK after it's successfully loaded. If there's an issue with the SDK (like it being blocked b/c you're in China), then the function will never get called and the login button will remain hidden.